### ISSUE TITLE
sources-list: are all repos listed correct?

### ISSUE THREAD

**Issue by star-buck** [netrunner-core/1803-issues#16]
![screenshot_20180302_120008](https://user-images.githubusercontent.com/840070/36900511-db5a7312-1e23-11e8-9ba9-3cc9c2952f9a.png)

- what is that unticked deb.netrunner.com for?
- and is the ticked dci.ds9.pub one correct (with nomad at the end)?

**Comment by llelectronics** [comment]
The not turned deb.netrunner.com is the backports repo apparently. 
The dci.ds9 is taking packages directly from our CI system. The nomad entry might be a copy and paste issue? (@shadeslayer or is it there on purpose?)

**Comment by star-buck** [comment]
Shouldnt both be the same url?
e.g. deb.netrunner.com?

**Comment by llelectronics** [comment]
In the final image it should as we don't want users to get our packages directly from dcis output.

**Comment by shadeslayer** [comment]
So this list is only partially correct. Let me explain from the top:

* I'm using the canonical snapshots.debian.org for users to decrease the overall cost to us to use our own mirror. We only want our own mirror for the CI
* The deb.debian.org entry is correct
* The deb.netrunner.com netrunner-backports entry is incorrect, I will fix this
* The dci.ds9.pub entry is on purpose since these are dev images, and in order to save disk space on the publishing server, I decided to use the live repos for now. I'll switch the entry to deb.netrunner.com starting next week.
* The skype repo is correct

**Comment by star-buck** [comment]
Okay, thanks fkr the clarify.
Lets get this changed next week first, so this weeks iso is RC, then final
isos will be correct to release any of the isos also as public if ready.

**Comment by star-buck** [comment]
*for the clarify, damn mobile...

**Comment by shadeslayer** [comment]
Sounds good to me :)

**Comment by shadeslayer** [comment]
One of the questions that now needs answering is this:

We currently use a Debian snapshot from the future. What this means is that updates will continue to get delivered till the 15th of March. If we release the ISO on the 12th, users get another 3 days of updates to their systems. Is this something we want? Or should I change the snapshot to be the one from the 12th and re spin ISO's?

**Comment by llelectronics** [comment]
I think we have to decide that shortly before release. Or to put it in other words. When we decide on a date to release we will have to set the Debian snapshot repo to either that date or the date of our last testing. Otherwise there might still slip something in breaking stuff. (at least this time we make this 0,01% possibility 0,0)

**Comment by star-buck** [comment]
did anyone think of switching the repos AWAY from dci?
![screenshot_20180311_183738](https://user-images.githubusercontent.com/840070/37256466-75143806-255b-11e8-83c6-7908e37403ac.png)

@shadeslayer  to quote:
"The dci.ds9.pub entry is on purpose since these are dev images, and in order to save disk space on the publishing server, I decided to use the live repos for now. I'll switch the entry to deb.netrunner.com starting next week."

**Comment by shadeslayer** [comment]
I fucked this one up, I didn't commit the change to git to change the repos, so the last iso had the live repos. Fixed in live build now.

**Comment by star-buck** [comment]
The "last" ISO was the public ISO, so your effort to change it now for new
builds is even counterproductive.

**Comment by llelectronics** [comment]
I suggest replacing the released ISO with the newer version though and maybe a short notice on the website.

**Comment by shadeslayer** [comment]
I agree, it's better to release a fixed ISO and also release a package update that fixes the issue.

**Comment by llelectronics** [comment]
Fix pushed for existing users with this https://github.com/netrunner-desktop/netrunner-desktop/commit/b2edf61f41d285fec4de9968a21065c52ee28871
